packages/react-emotion/src/index.js

```diff
@@ -85,7 +85,7 @@ const createStyled = (tag, options: { e: string, label: string }) => {
       render() {
         const { props, state } = this
         this.mergedProps = omitAssign(testAlwaysTrue, {}, props, {
-          theme: (state !== null && state.theme) || props.theme || {}
+          theme: (state && state.theme) || props.theme || {}
```

Comment:
Could this be `Object.assign({}, props.theme, state.theme)`?

Reply:
Not sure on the implications of using Object.assign as far as browser support goes.

Reply:
The `undefined`/`null` check would still be necessary with object.assign. I don't think Object.assign has any support in IE.